data-collection/data-collection/AppFiles.swift

```diff
@@ -15,29 +15,26 @@
 import Foundation
 import ArcGIS
 
-/// This class is used for creating and removing directories and items needed for the app in the device's file documents directory.
-class AppFiles {
-    
-    private var fm: FileManager { return FileManager.default }
-    
+/// This class extension is used to creating and removing directories and items needed for the app in the device's file documents directory.
```

Comment:
As far as I know, there's no benefit to using a documentation comment on extensions.

```suggestion
// This class extension is used to creating and removing directories and items needed for the app in the device's file documents directory.
```

Reply:
Good catch.